src/sentry/issues/escalating.py

```diff
@@ -331,7 +331,11 @@ def manage_issue_states(
             add_group_to_inbox(group, GroupInboxReason.ESCALATING, snooze_details)
             record_group_history(group, GroupHistoryStatus.ESCALATING)
             issue_escalating.send_robust(
-                project=group.project, group=group, event=event, sender=manage_issue_states
+                project=group.project,
+                group=group,
+                event=event,
+                sender=manage_issue_states,
+                was_until_escalating=True if data and activity_data else False,
```

Comment:
Could you please document the docstring a bit better indicating what `activity_data` is?

Would `previous_state` instead of naming it `was_until_escalating ` and a string be better than a bool? Or does it have to be a bool?

Reply:
Thank you for the suggestion! I considered that too, but in this function I can really only determine if the previous state was `until_escalating` without making more changes. I didn't want most of the entries to be `null`, as this would be confusing